libs/blocks/accordion/accordion.css

```diff
@@ -144,3 +144,82 @@ dl.accordion {
 .dark .accordion dt button:hover .accordion-icon::after {
   background: var(--color-gray-100);
 }
+
+/* Editorial Variation  */
+.accordion-media {
+  display:  none;
+}
+
+.accordion-media > div {
+  position: relative;
+  display: none;
+  animation-duration: .1s;
+  animation-name: slide-left;
+}
+
+/***** Slide to the Left *****/
+.slide-left {
+  animation: 1s slide-left;
+}
+@keyframes slide-left {
+  from {
+    margin-right: -100%;
+  }
+  to {
+    margin-right: 0%;
+  }
+}
```

Comment:
Loving the animation details!

I almost feel a simple fade would be better suited. 
```
@keyframes fade-in {
  0% {
    opacity: 0;
  }
  100% {
    opacity: 1;
  }
}
```
I'll leave it up to you.